studio/credentials/credentials.py

```diff
@@ -102,14 +102,14 @@ def get_fingerprint(self) -> str:
         if self.type is None and self.key is None and\
             self.secret_key is None:
             return ''
-        id: str = "{0}::{1}::{2}::{3}::{4}::{5}"\
+        id_str: str = "{0}::{1}::{2}::{3}::{4}::{5}"\
```

Comment:
Why is the type added to the variable name , is this a convention for pylint ?

Reply:
No. It could be id_xyz for that matter, just not "id", which is considered "reserved identifier" in Python.